samples/msal-node-samples/nextjs-rsc-ts/src/actions/auth.ts

```diff
@@ -0,0 +1,31 @@
+"use server";
```

Comment:
While it's great to see how this is done with Server Actions, given they're in alpha not sure if it's wise to showcase them in this sample? Maybe have a separate sample that showcases their use with a warning to be explicit about their early-stage development.

Source: https://nextjs.org/docs/app/building-your-application/data-fetching/server-actions

Reply:
I agree it might be a little early for server actions. But I do feel they are an integral part to the RSC story, so I'd still like them to be a part of this sample.

But I have added some sample buttons that utilizes Route Handlers as well 🙂